File: go/remove-item-from-array.go
package main

import "fmt"

func main() {
    cars := [3]string{"Ford", "Volvo", "BMW"}

    fmt.Println(cars)

    newArr := [len(cars)-1]string{}
    var j int = 0
    for i := 0; i < len(cars); i++ {
        if cars[i] != "Volvo" {
            newArr[j] = cars[i]
            j++
        }
    }

    fmt.Println(newArr)
}

File: rust/remove-item-from-array.rs
fn main() {
    const cars:[&str; 3] = ["Ford", "Volvo", "BMW"];

    println!("{:?}", cars);

    let mut newArr:[&str; cars.len()-1] = ["A"; cars.len() - 1];
    let mut j = 0;
    for i in 0..cars.len() {
        if cars[i] != "Volvo" {
            newArr[j] = cars[i];
            j = j + 1;
        }
    }

    println!("{:?}", newArr);
}
